packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js

```diff
@@ -0,0 +1,200 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const CODE = 'LESMIS_gross_enrolment_rate';
+
+const REPORT_CONFIG = {
+  fetch: {
+    dataElements: [
+      'ger_district_pe_t',
+      'ger_district_pe_f',
+      'ger_district_pe_m',
+      'ger_district_lse_t',
+      'ger_district_lse_f',
+      'ger_district_lse_m',
+      'ger_district_use_t',
+      'ger_district_use_f',
+      'ger_district_use_m',
+    ],
+    aggregations: [
+      {
+        type: 'MOST_RECENT',
+        config: {
+          dataSourceEntityType: 'sub_district',
+        },
+      },
+    ],
+  },
+  transform: [
+    {
+      transform: 'select',
+      "'name'":
+        "translate($row.dataElement, { ger_district_pe_t: 'Primary', ger_district_pe_f: 'Primary', ger_district_pe_m: 'Primary', ger_district_lse_t: 'Lower Secondary', ger_district_lse_f: 'Lower Secondary', ger_district_lse_m: 'Lower Secondary', ger_district_use_t: 'Upper Secondary', ger_district_use_f: 'Upper Secondary', ger_district_use_m: 'Upper Secondary' })",
+      '...': '*',
+    },
+    'keyValueByDataElementName',
+    {
+      transform: 'aggregate',
+      name: 'group',
+      '...': 'last',
+    },
+    {
+      transform: 'select',
+      "'Male'": 'sum([$row.ger_district_pe_m, $row.ger_district_lse_m, $row.ger_district_use_m])',
+      "'Female'": 'sum([$row.ger_district_pe_f, $row.ger_district_lse_f, $row.ger_district_use_f])',
+      "'Total'": 'sum([$row.ger_district_pe_t, $row.ger_district_lse_t, $row.ger_district_use_t])',
+      '...': ['name'],
+    },
+    {
+      transform: 'select',
+      "'GPI'": '$row.Female/$row.Male',
+      '...': '*',
+    },
+    {
+      transform: 'select',
+      "'sort_order'":
+        "translate($row.name, { 'Primary': 1', 'Lower Secondary': '2', 'Upper Secondary': '3' })",
+      '...': '*',
+    },
+    {
+      transform: 'sort',
+      by: '$row.sort_order',
+    },
+    {
+      transform: 'select',
+      '...': ['name', 'Male', 'Female', 'Total', 'GPI'],
+    },
+  ],
+};
+
+const FRONT_END_CONFIG = {
+  name: 'Gross Enrolment Rate - GER (by grade and gender, GPI)',
+  type: 'chart',
+  chartType: 'composed',
+  xName: 'Level of Education',
```

Comment:
```suggestion
  xName: 'Level of Education',
  yName: 'Rate (%),
```

Reply:
LESMIS currently doesn't display `yName` at all, but putting the `yName` at this level applies it to _both_ yAxis in Tupaia, currently I have the `yName` under the `GPI` and `Male` individual configs for the sake of that

![image](https://user-images.githubusercontent.com/69437175/123036350-0f531d00-d441-11eb-9029-7eaf4b583ffd.png)

Reply:
Aaah right that makes sense! I'll fix mine up to be the same